deploy/webhook-certs.yaml

```diff
@@ -1,7 +1,7 @@
+  key.pem: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBdmxWbFJrcGVrT08rNVhJbmFMSkFTUjdVWXlSOU5LN3lTZ2NuWDBydjN0V1IyeHQyCmljb1dQeFNqc0NEZllGRUY4a25LZFhLQnZNeFR0TXpUM2JSUmhzcGRva2s0a1dBVlFYSnllSE9YY0htMUZxUXIKRVhNOGdXNDFQWTlvQlAxbzhtWjkxbHd1QkJIVVFRT3hjUEMvQm5ibnVJc1pKaW81ekJGY290NTA4MGpWSTFKRQprYzFrQXJHYlZVRmdNVy9nMjQ3WFVTay93NSt3am8yZ1J1b1YyRUpqZitkOWREbWIvblN2aU1VWnlIWnR5NGdzCk9kWmdGbmxDdGxGVkJBdDVhcXNJRWhNZ1pheFJ0RVBNZGc2Mk40OHFwNzcwdjZLRFRrY3ZWaUFFRWIyaC9TUkEKQ3o0T0JtQjlDRVB5VDZWdWt6RXJ1a2xGK2NMTWhWWU5VZXg1bHdJREFRQUJBb0lCQUZuVHo3STIweDZxZmFPbwpGblZRK3dvSFpORk5uVkhKNTB1TXlNWXVoYnF6TE4wdHZ1b1UyQjc1VEsrSk5DTXUyaEdaUmRYY0x1Szl4QnVtCk42K0F0S0ZheklvVTZIK2p5eVJTTVRncnhhTXRyWXQ3KzErRzlWMVllQVFQdEZYMml4c3J4K1B0S1VKK2I2QW4KY0EwUGNmaEpFZCtFdGwvT3VleFhuVks0anBWcVQrRk55UFYyVVRFUnV6cWhVY0dCc0p0bnVkZkpWMFdRZXRGVAordUxCWTFtM3VuY2hHWlo1cnlML0NWYVQxc1NiMk9LM2VMUTRaWmZ3UFFHc3pFMytaU2hJWnRlcE94QzBVbE5yCnNNOXFGODBtN2hWNU5hT0FHZXkxZlRDd0VOTFhIaENKMFpnbGd2ajN3RWxCeUtsTFVSenJjOW9mRFY5SUl3SGIKTFQrWXlGRUNnWUVBM2FNMGh3alhmV0xnS1BRSVBVSGE2QTJvZ0E2cjRTQ25lYTJpUjMrZjVWZlBFOTg2SXBTKwp4Qk9QVzdqS3ZiaVl1WGRua0dSa010b0VmTHZyOW1HRWYwUTFTdGtDSjEvbjVxSG42NmJ5M3VLb1ZkZlpKYmtUCld6cWpFWHVRRGpzZ01HUnA0aDJ0cVdQNFZ5aEphSktBRTc1bUszT0dNZHpXMEdaRXY0MjFuOHNDZ1lFQTI5ZTgKbGUyK2ltQnRXR05GY3ZjYUQ2ZlhLdUpCSDJsSVhRQyt2L2VLZjd2M3lZbUJmN0VGS3ZHL1hnL3VWS1J0dlp4SAorazZrUk9lblp4Z1lOc1I1QTFUeUlTT3R0ajhINmppcDRGcFpjWGUxS1JsbXIyTFFaaEFhZmRpWlFEanBCUUlvCmcwb2ZRZFcvOURTL3BDNjBBbk00ejRlazhGYU1jOER0SUE1S2UrVUNnWUJiRnpPRWVCN1FtcTJVUjArSGxhUloKa1J0Nm5nMElOcnArS04yMFV3Q3RhcHhpQjNIR09xbjR2aTFDUGoyZElkd3JCYWNiMGhiWVM0QmluMzJJUW1ZSgpSRWcrM2RlanhTZnpRd0dsZVhVQ0k5ZVppRi9ud0J2emQ3TURnV3dnWFM2S2xRcHJjMVVuZEQzaDNGM25TNmExCjlwL1pKZDYyYVpRa0RKUWV3aWd1WndLQmdDNVF1aU5nRGdkN0h5NUM0OHUvc1ZiaEJseTdnL3JKSkh5cmZvUlUKODNITHVQMVZvZG0wQXFHMGdwdWZnWmVGZUxVOHFwRmFaaE02TnJOcWlSZG90eEh6b0h0STEvMFFOR3VzK21VUQp3ODA2NmdpNW1EK1U4dEM5emNiN2pmWFF4NDdOa0psd1VUdCtlODFJRzZQTVR4SkJFaFB1d05uOWNlT2xIby9sClV0M2RBb0dCQU10SHduTWwxdUZWbW4yME03NmlDTytsNVhSZjNjam5QV1B3ZlJyTXprWVlqTXNUaTFNMmFOZUoKd05IbTVOWG5hTm5BSWhjaThKc3RjRXdrVzNZa1FNdmpTeStaRW9obSs1dVVGTkladWtDVDhoWWoyWVRXaWZWbQpEZ2NkMEhRcW1NUEUvQTFORkxHd3dRakxUUjcvNTFhUU13YlZaUHY2cWlBZUxDMnVCTmVxCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==
 kind: Secret
 metadata:
   creationTimestamp: null
```

Comment:
I think there is no need to update this file

Reply:
You're right, it's just in case somebody runs `make deploy` without having generated the certs.

This is just temporary, I think it would make sense to somehow get rid of these manifests in favor of helm to avoid maintaining two sets of manifests. We could still generate manifests using `helm template`.